src/ui/component/button/button.scss

```diff
@@ -86,6 +86,39 @@
         border-color: themed('secondary-disabled');
       }
 
+      &:hover {
+        background-color: unset;
+        color: themed('secondary-color-variant-1');
+      }
+
+      svg {
+        path {
+          fill: themed('secondary-color');
+        }
+      }
+    }
+
+    &.quaternary-turquoise {
+      @include button(
+        $content-color: themed('primary-color-variant-3'),
+        $background-color: unset,
+        $box-shadow: none,
+        $hover-color: themed('primary-color-variant-2')
+      );
+      border-radius: 6px;
+      border: 1px solid themed('primary-color-variant-3');
+
+      &.disabled {
+        background-color: unset;
+        color: themed('secondary-disabled');
+        border-color: themed('secondary-disabled');
+      }
+
+      &:hover {
+        background-color: unset;
```

Comment:
Same here, you can omit `background-color` props since it's the same as the `quaternary-turquoise` class.

Reply:
Hmm I think I cannot remove this one because then when hovered over, the button gets a background color and according to the design system only the border and content color should change (basically no background color, it should stay outlined)

Reply:
You are right, I forgot this details! You can keep it as is.